python/src/fds/protobuf/stach/extensions/v2/StachUtilities.py

```diff
@@ -47,3 +47,33 @@ def get_value(val):
         else:
             return val
 
+    @staticmethod
+    def decompress_column(column):
+        """
+        This method is used to decompress a stach response column
+        :param column: The column to be decompressed
+        :return decompressed_column: returns the decompressed column
+        """
+
+        if "ranges" in column:
+            decompressed_column = column["values"].copy()
+            for index, length in column["ranges"].items():
+                index = int(index)
+                decompressed_column[index:index] = [decompressed_column[index]] * (length - 1)
+            return decompressed_column
+        else:
+            return column["values"]
+
+    @staticmethod
+    def decompress_page(result_pages):
```

Comment:
Can we call this method as `decompress_all_columns(package)`? Page seems can be a new terminology for users of this package.